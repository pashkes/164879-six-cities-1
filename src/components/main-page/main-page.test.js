import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import MainPage from "./main-page.jsx";

configure({adapter: new Adapter()});

it(`main page renders correctly`, () => {
  const catalog = shallow(
      <MainPage
        isAuthorizationRequired={false}
        authorization={{}}
      />
  );
  expect(catalog).toMatchSnapshot();
});
