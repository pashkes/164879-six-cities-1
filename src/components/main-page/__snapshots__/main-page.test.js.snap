// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main page renders correctly 1`] = `
<Layout
  pageClasses={
    Array [
      "page",
      "page--gray",
      "page--main",
    ]
  }
  title="Six cities"
>
  <ConnectFunction />
</Layout>
`;
